### PR TITLE
LOG-1911: Enable opt-in of multiline error detection

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR enables opt-in of multiline error detection to address admin's understanding of associated impact

### Links
* https://issues.redhat.com/browse/LOG-1911

### CODE REVIEW

**Comment by jcantrill** [comment]
/assign @alanconway 

cc @anpingli

File: apis/logging/v1/cluster_log_forwarder_types.go [hunk 1]
```diff
@@ -232,6 +232,11 @@ type PipelineSpec struct {
 	// +kubebuilder:validation:Enum:=json
 	// +optional
 	Parse string `json:"parse,omitempty"`
+
+	// EnableMultilineErrorDetection enables multiline error detection of container logs
+	//
+	// +optional
+	EnableMultilineErrorDetection bool `json:"enableMultilineErrorDetection,omitempty"`
```
Comment (alanconway) [inline apis/logging/v1/cluster_log_forwarder_types.go:239]: Suggest "detectMultilineErrors: true/false" - sorry for the nit, but I think it's important to keep the API as brief as possible without sacrificing clarity.

**Review by alanconway — COMMENTED** [review]
Name change suggestion,  otherwise LGTM.

**Review by alanconway — APPROVED** [review]
/lgtm

**Pull request merged**